.travis.yml

```diff
@@ -26,7 +26,6 @@ before_script:
   # test to see if the db is an issue
   - mysql -e 'CREATE DATABASE probabledev;'
   - mysql -e 'CREATE DATABASE probabletest;'
-  - mysql -e "SET PASSWORD FOR 'travis' = PASSWORD('probable');"
   - mysql -e "GRANT ALL PRIVILEGES on probabledev.* TO 'travis';"
```

Comment:
Do you need a development database to run integration tests?

Reply:
Yes, the feature tests run off of the development database